variables.tf

```diff
@@ -285,3 +285,19 @@ variable "vcva_deployment_option" {
   type        = string
   default     = "small"
 }
+
+variable "update_esxi" {
+  description = "if true update the ESXi version before proceeding to vCenter installation"
+  type        = bool
+  default     = false
+}
+
+variable "esxi_update_filename" {
+  description = <<-EOF
+  The specific update version that your servers will be updated to.
+  Note that the Equinix Metal portal and API will still show ESXi 6.5 as the OS but this script adds a tag with the update filename specified below.
+  You can check all ESXi update versions/filenames here: https://esxi-patches.v-front.de/
+  EOF
+  type        = string
+  default     = "ESXi-7.0U3d-19482537-standard"
```

Comment:
Looks like latest is now ESXi-7.0U3m-21686933-standard
https://esxi-patches.v-front.de/ESXi-7.0.0.html

(just noting this as I take this PR for a test drive with some of the other open PRs all merged together)

Reply:
It looks like I'm running into 5m timeouts with the network config resources regardless of this PR.

`null_resource.apply_esx_network_config[0]` 
`null_resource.apply_esx_network_config[1]`